addons/payment/models/payment_transaction.py

```diff
@@ -1047,7 +1047,7 @@ def _get_received_message(self):
             )
             if self.payment_id:
                 message += _(
-                    "\nThe related payment is posted: %s",
+                    "<br />The related payment is posted: %s",
```

Comment:
```suggestion
                    "<br/>The related payment is posted: %s",
```
Also, it could be a good idea to take the `<br/>` out of the translated string to avoid confusing translators:
```py
                message += '<br/>' + _(
                    "The related payment is posted: %s",
                    self.payment_id._get_payment_chatter_link(),
                )
```